chia/wallet/puzzles/nft_ownership_transfer_program_one_way_claim_with_royalties_new.clvm

```diff
@@ -0,0 +1,114 @@
+(mod
+  (
+    P2_DELEGATED_PUZZLE_OR_HIDDEN_PUZZLE_MOD_HASH
+    SINGLETON_STRUCT
+    ROYALTY_ADDRESS
+    TRADE_PRICE_PERCENTAGE
+    SETTLEMENT_MOD_HASH
+    CAT_MOD_HASH
+    Current_Owner  ; Truth
```

Comment:
Why only first letters are in uppercase? Is it because they are curried in the higher level?